index.js

```diff
@@ -34,6 +34,11 @@ app.use(express.json());
 
 app.use('/api', usersRouter);
 app.use('/api', authRouter);
+
+app.use(passport.authenticate('jwt',
```

Comment:
I put the jwt auth inside the actual router, otherwise we will get a 401 unauthorized when we are expecting to see another error. Check out the itinerary router for an example.